layouts/partials/favicons.html

```diff
@@ -0,0 +1,22 @@
+{{ if isset .Site.Params "favicons" }}
+{{ if .Site.Params.favicons.enable }}
+<!-- Due to the nature of Favicons, it is best to put them in the root directory
+Rather than in a seperate folder, but will work from in a folder in most cases. -->
+<link rel="apple-touch-icon" sizes="180x180" href="{{ .Site.BaseURL }}apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="{{ .Site.BaseURL }}favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="194x194" href="{{ .Site.BaseURL }}favicon-194x194.png">
+<link rel="icon" type="image/png" sizes="192x192" href="{{ .Site.BaseURL }}android-chrome-192x192.png">
```

Comment:
Favicon `android-chrome-192x192.png` is missing in the static directory.

In any case, should this one be loaded here or in the `manifest.json` file like the rest of Android favicons?